images/cellregmap/requirements.txt

```diff
@@ -0,0 +1,2 @@
+click
+pandas
```

Comment:
Do you want to add the remainder of packages you need here? For example, 
- cellregmap
- numpy
- scanpy
- pandas + pandas_plink
- xarray
- limix

You should look to fix all versions (eg: `cellregmap == 1.0.0`)

Reply:
should've done! Including versions